reference/upgrades.md

```diff
@@ -1,5 +1,96 @@
 ## Upgrading Mayastor
 
-Mayastor v2.0.0 does not support upgrade from previous versions. The installation of the new version should be preceded by the removal of any earlier version.
+Upgrade to the latest Mayastor version is supported only from 2.1.0 or later. To upgrade from a previous version, visit [Manual steps to upgrade Mayastor](manual-steps-to-upgrade-mayastor). 
+
+### Supported upgrade paths
+
+- From 2.0.x to 2.1.0
+
+{% hint style="info" %}
+The process of upgrade utilises the [Mayastor Kubectl Plugin](https://mayastor.gitbook.io/introduction/advanced-operations/kubectl-plugin) and is non-disruptive only for volumes with more than one healthy replicas.
+{% endhint %}
+
+To upgrade all the Mayastor components of a cluster, execute:
```

Comment:
To upgrade Mayastor deployment on the Kubernetes cluster,

Reply:
Updated the text.